### PR TITLE
issue #1202: docs(help examples): adding template and first examples …

### PR DESCRIPTION
**Pull request by igaskin**
…for the app command

shameless ripoff of kubectl example templating

### CODE REVIEW

File: util/templates/normalizers.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
Comment (alexec) [inline util/templates/normalizers.go]: Please remove the copyright notice.

File: Gopkg.lock [hunk 2]
```diff
@@ -17,6 +17,14 @@
   revision = "d216395917cc49052c7c7094cf57f09657ca08a8"
   version = "v3.0.0"
 
+[[projects]]
+  branch = "master"
+  digest = "1:6b250e53b3b7b9abd7e62f55875c234d2f8b77b846bff200fecafa2dc09af09a"
+  name = "github.com/MakeNowJust/heredoc"
```
Comment (alexec) [inline Gopkg.lock]: Licence is acceptable: https://github.com/MakeNowJust/heredoc/blob/master/LICENSE

File: util/templates/normalizers.go (hunk 1 above)
Comment (alexec) [reply util/templates/normalizers.go]: What about checking for vulns/malicious code? To discus.

File: Gopkg.lock (hunk 2 above)
Comment (igaskin) [reply Gopkg.lock]: 👍 going to remove this library since it wasn't used.

**Comment by igaskin** [comment]
@alexec The unit tests failed because an index out of range on `TestManipulateApplicationResources`.  Is this something I should be correcting before merging this PR?  I'm almost certain that my changes should not have effected this test case.

**Comment by alexmt** [comment]
@igaskin you are right about the test. Looks like it is flaky. will take a look into it

**Review by alexmt — APPROVED** [review]
LGTM

**Pull request merged**